contracts/src/RatesManager.sol

```diff
@@ -83,10 +84,16 @@ contract RatesManager is Initializable, AccessControlUpgradeable, UUPSUpgradeabl
         }
 
         allRates.push(
-            Rates({messageFee: messageFee, storageFee: storageFee, congestionFee: congestionFee, startTime: startTime})
+            Rates({
+                messageFee: messageFee,
+                storageFee: storageFee,
+                congestionFee: congestionFee,
+                startTime: startTime,
+                targetRatePerMinute: targetRatePerMinute
+            })
```

Comment:
nit: I believe we're storing the target rate in Rates to help with the calculations when attesting to a report, is the assumption correct?
I'm ok with this, it just seems a bit off at first as the target rate could be a different entity that changes independently from the fees.

Reply:
I thought about giving the target rate its own smart contract. But having them all update in sync (at the same start time) makes the system a little cleaner and easier to model. One "fee epoch" instead of multiple.